hub-database.sh

```diff
@@ -31,8 +31,9 @@ echo "Database host: $targetDatabaseHost"
 ## Until we need another requirement from customers that they need to make a change on pg_hba.conf
 ## trigger this each time when starting a container.
 ## (needs to be kept in sync with 2-hub-setup.sh)
-if [ -d "$PGDATA" ] && [ -s "$PGDATA/PG_VERSION" ];
+if [ -d "$PGDATA" ] && [ -s "$PGDATA/$PG_VERSION" ];
```

Comment:
This change is not correct.  PG_VERSION is a file that is created in PGDATA by initdb.  This test is checking that the data directory exists and that initdb has been run.

Reply:
Change reverted, should be using "PG_VERSION" in all instances now